### DISCUSSION TITLE
Setting child gap doesn't have effect

### DISCUSSION THREAD

**Discussion by Lotus49b** [freeplane/freeplane#380] [Help] [answered]
Hi, I'm trying to set a vertical distance between level 2 nodes (in this particular case) but the change has no effect.

Also, the change to the parameter (changing "node shape/child gap" from 2pt to 20pt) simply resets back to default after I apply the change to this map.

Any advise on how to make the change stick? 

TIA

**Comment by EdoFro** [comment]
hi,
did you change it in a node or in a node style?
if in a style: is your map linked to a template map? (follows template styles?)

**Reply by EdoFro** [reply]
maybe your map styles get updated from a template map, where the child gap is still 2 pt.

**Reply by Lotus49b** [reply]
Hi
I was making the changes in "node style" and it is linked to a template. 

Was just checking the map against your comments and suddenly the changes are now taking effect!? Strange. Maybe because I closed the app overnight and re-opened the map?

Thanks for taking the time to respond, really appreciate that.

**Comment by dpolivaev** [comment]
If you want anyone to be able to understand the behavior you need to attach the (zipped) example mind map file and reference concrete nodes in your question. The observed behavior can have multiple reasons. Guesses without complete input are not effective.

**Reply by Lotus49b** [reply]
Thanks Dimitry, and noted.

And thank you for a great application! It's become an essential part of my workflow, especially now that I can use it to outline thoughts and narratives, and then move those as markdown documents into Obsidian.

**Comment by petera703** [comment, marked as answer]
Came here to see if I could help because it shows as unanswered:
![image](https://user-images.githubusercontent.com/40151458/170346898-2dd8a187-cc04-46a8-8a8e-cd7000f3e580.png)
But I can see that it is resolved.

**Reply by Lotus49b** [reply]
Thanks, it's not actually resolved but I'll mark it as "answered" so that more people don't view it.
